Microsoft.Toolkit.Uwp.UI.Controls/Graph/SharePointFiles/SharePointFiles.cs

```diff
@@ -0,0 +1,558 @@
+                        await LoadFiles(rootDriveItem.Id);
+                        _back.Visibility = Visibility.Collapsed;
+                    }
+                }
+            }
+            catch
+            {
+            }
+        }
+
+        private async Task LoadFiles(string driveItemId, int pageIndex = 0)
+        {
+            IsDetailPaneVisible = false;
+            if (!string.IsNullOrEmpty(_driveId))
+            {
+                try
+                {
+                    _cancelLoadFile.Cancel(false);
+                    _cancelLoadFile.Dispose();
+                    _cancelLoadFile = new CancellationTokenSource();
+                    _list.Items.Clear();
+                    _download.Visibility = Visibility.Collapsed;
+                    _share.Visibility = Visibility.Collapsed;
+                    _delete.Visibility = Visibility.Collapsed;
+                    QueryOption queryOption = new QueryOption("$top", PageSize.ToString());
+                    Task<IDriveItemChildrenCollectionPage> taskFiles = _graphClient.Drives[_driveId].Items[driveItemId].Children.Request(new List<Option> { queryOption }).GetAsync(_cancelLoadFile.Token);
+                    IDriveItemChildrenCollectionPage files = await taskFiles;
+                    if (!taskFiles.IsCanceled)
+                    {
+                        _list.Items.Clear();
+                        foreach (DriveItem file in files)
+                        {
+                            _list.Items.Add(file);
+                        }
+
+                        _nextPageRequest = files.NextPageRequest;
+                        HasMore = _nextPageRequest != null;
+                        _upload.Visibility = Visibility.Collapsed;
+                        if (_driveItemPath.Count > 1)
+                        {
+                            IDriveItemPermissionsCollectionPage permissions = await _graphClient.Drives[_driveId].Items[driveItemId].Permissions.Request().GetAsync();
+                            foreach (Permission permission in permissions)
+                            {
+                                if (permission.Roles.Contains("write") || permission.Roles.Contains("owner"))
+                                {
+                                    _upload.Visibility = Visibility.Visible;
+                                    break;
+                                }
+                            }
+                        }
+                        else
+                        {
+                            _upload.Visibility = Visibility.Visible;
+                        }
+                    }
+
+                    if (_list.Items.Count > 0)
+                    {
+                        _list.SelectedIndex = 0;
+                    }
+                }
+                catch (Exception)
+                {
+                }
+            }
+        }
+
+        private async Task LoadNextPage()
```

Comment:
Rename by `LoadNextPageAsync` for consistency

Reply:
@Odonno done